doc/api/cli.md

```diff
@@ -815,6 +815,22 @@ changes:
 Specify the `module` containing exported [module customization hooks][].
 `module` may be any string accepted as an [`import` specifier][].
 
+### `--experimental-minimum-coverage`
+
+<!-- YAML
+added: v22.0.0
+-->
+
+> Stability: 1 - Experimental
+
+To Specify the minimum percentage threshold value for code coverage.
+Used with `--experimental-test-coverage` flag. If the minimum threshold
+value is not met, the test will exit with status code 1.
```

Comment:
The documentation doesn't say anything about what coverage metric is meant here.

Reply:
Indeed; typically in code coverage there are 4 metrics (statements remains missing), and all 4 are independently able to be set to a threshold.